studio/components/to-be-cleaned/ModalsDeprecated/InviteMemberModal.tsx

```diff
@@ -0,0 +1,155 @@
+import { createContext, useState, useEffect } from 'react'
+import { isNil } from 'lodash'
+import { observer, useLocalObservable } from 'mobx-react-lite'
+import { post } from 'lib/common/fetch'
+import { Button, IconKey, IconMail, Input } from '@supabase/ui'
+import { Modal } from '@supabase/ui'
+
+import { API_URL } from 'lib/constants'
+import { useOrganizationDetail, useStore } from 'hooks'
+import { toJS } from 'mobx'
+import { Member } from 'types'
+
+/**
+ * Endpoints (delete when finished)
+ *
+ * Delete invite
+ * curl -X DELETE "http://localhost:8080/platform/organizations/:slug/members/invite?invited_id=:id" -H "Authorization: Bearer access_token_jwt"
+ *
+ * Get all pending invites
+ * curl -X GET "http://localhost:8080/platform/organizations/:slug/members/invite" -H "Authorization: Bearer access_token_jwt"
+ *
+ * Send an invitation
+ * curl -X POST "http://localhost:8080/platform/organizations/slug/members/invite" -H "Content-Type: application/json" -d '{"invited_email": "", "owner_id": ""}' -H "Authorization: Bearer ..."
+ *
+ * Verify an invitation
+ * curl -X GET 'http://localhost:8080/platform/organizations/slug/members/join?token=XXX'
+ *
+ */
+
+/**
+ * Modal to invite member to Organization
+ *
+ * @param {Object}          organization          // organization detail
+ * @param {Array}           members               // organization members list
+ * @param {Object}          user                  // current user detail
+ */
+
+const PageContext = createContext(null)
+
+function InviteMemberModal({ organization, members = [], user }: any) {
+  const PageState = useLocalObservable(() => ({
```

Comment:
Let's move away from using useLocalObservable, and use Form from supabase/ui instead - lemme know if you need help with this one

Reply:
Updated, thanks!  <Form> is super nice!